db/migrate/20180913135147_ingest_sierra_code_zcode_matches2.rb

```diff
@@ -0,0 +1,5 @@
+class IngestSierraCodeZcodeMatches2 < ActiveRecord::Migration
+  def up
+    Rake::Task['ingest:overwrite_sierra_code_zcode_matches'].invoke('data/public/sierra_code_zcode_matches.csv')
```

Comment:
Am I correct in reading this as a destructive change?

Reply:
@daryanypl Yes, when we overwrite the table, all the codes get new primary IDs, but that won't have an impact because we don't reference the primary IDs anywhere; we just send the code in the request to the Patron Creator Service.